packages/terra-form-checkbox/src/Checkbox.scss

```diff
@@ -0,0 +1,153 @@
+:local {
+  .checkbox {
+    color: var(--terra-form-checkbox-font-color, rgb(0, 0, 0));
+    display: block;
+    font-family: inherit;
+    margin-left: var(--terra-form-checkbox-container-margin-left, none);
+    margin-right: var(--terra-form-checkbox-container-margin-right, 2rem);
+    margin-top: var(--terra-form-checkbox-container-margin-top, 0.625rem);
+
+    /* Hitbox and container focus style will only surround checkbox + label, instead of the div block */
+    /* Padding puts space for container focus style */
+    .inner-div {
+      display: inline-block;
+      padding-left: var(--terra-form-checkbox-inner-div-padding-left, 0.214rem);
+      padding-right: var(--terra-form-checkboc-innter-div-padding-right, 0.214rem);
+    }
+
+    /* Hide label if isLabelHidden true */
+    .label-hidden {
+      clip: rect(0 0 0 0);
+      cursor: default;
+      visibility: hidden;
+    }
+
+    /* stylelint-disable selector-max-compound-selectors */
+    .label {
+      cursor: pointer;
+      display: flex;
+      font-size: var(--terra-form-checkbox-font-size, 1rem);
+      line-height: 1.42em;
+      padding-left: var(--terra-form-checkbox-padding-left, none);
+      position: relative;
+    }
+
+    /* Native checkbox */
+    .native-input {
+      border: var(--terra-form-checkbox-native-input-border, none);
+      clip: var(--terra-form-checkbox-native-input-clip, none);
+      cursor: var(--terra-form-checkbox-native-input-cursor, pointer);
+      font-size: var(--terra-form-checkbox-native-input-font-size, inherit);
+      height: var(--terra-form-checkbox-native-input-height, none);
+      margin-left: var(--terra-form-checkbox-native-input-margin-left, 0.071rem);
+      margin-right: var(--terra-form-checkbox-native-input-margin-right, 0.143rem);
+      overflow: var(--terra-form-checkbox-native-input-overflow, none);
+      padding: var(--terra-form-checkbox-native-input-padding, none);
+      position: absolute;
+      width: var(--terra-form-checkbox-native-input-width, none);
+    }
+
+    .inner-span {
+      margin-left: var(--terra-form-checkbox-inner-span-margin-left, 1.4em);
+    }
+
+    /* Custom checkbox */
+    .native-input + .inner-span::before {
+      background-color: var(--terra-form-checkbox-background-color, none);
+      background-image: var(--terra-form-checkbox-background-image, none);
+      background-position: center center;
+      background-repeat: no-repeat;
+      border-color: var(--terra-form-checkbox-border-color, none);
+      border-radius: var(--terra-form-checkbox-border-radius, none);
+      border-style: var(--terra-form-checkbox-border-style, none);
+      border-width: var(--terra-form-checkbox-border-width, none);
+      content: '';
+      cursor: pointer;
+      display: inline-block;
+      height: var(--terra-form-checkbox-height, none);
+      left: 0;
+      margin-top: var(--terra-form-checkbox-margin-top, none);
+      position: absolute;
+      top: 0;
+      visibility: visible;
```

Comment:
`visible` is the default value for `visibility`. You shouldn't need to set it here.